tests/js/spec/views/performance/trends.spec.jsx

```diff
@@ -370,8 +370,10 @@ describe('Performance > Trends', function () {
     );
 
     for (const trendFunction of TRENDS_FUNCTIONS) {
-      selectTrendFunction(wrapper, trendFunction.field);
       await tick();
+      wrapper.update();
+
+      selectTrendFunction(wrapper, trendFunction.field);
```

Comment:
Should this be before the tick + update?

Reply:
Actually I should do both to be safe, I had the one above for first run...